shells/backends/aws_tf_backend/shell-definition.yaml

```diff
@@ -0,0 +1,85 @@
+tosca_definitions_version: tosca_simple_yaml_1_0
+
+metadata:
+  template_name: Aws Tf Backend
+  template_author: Anonymous
+  template_version: 0.1.0
+  template_icon: shell-icon.png
+
+description: >
+  TOSCA based resource shell
+
+imports:
+  - cloudshell_standard: cloudshell_resource_standard_2_0_3.yaml
+
+node_types:
+
+  vendor.resource.Aws Tf Backend:
+    derived_from: cloudshell.nodes.GenericResource
+    properties:
+      Bucket Name:
+        description: The name of the bucket to be used in order to save the state file
+        type: string
+        tags: [ setting, user_input ]
+      Region Name:
+        description: The region in which the bucket resides
```

Comment:
Need to add that it's mandatory regarding if Cloud Provider is specified

Reply:
It`s always mandatory...I don`t want the region to be picked up from the clp